js/src/forum/components/ChangeEmailModal.js

```diff
@@ -121,8 +121,8 @@ export default class ChangeEmailModal extends Modal {
   }
 
   onerror(error) {
-    if (error.status === 401) {
-      error.alert.props.children = app.translator.trans('core.forum.change_email.incorrect_password_message');
+    if (error.status === 403) {
```

Comment:
Why is the error code changing here? Was it incorrect?

Reply:
Yep, 403 is what's returned when the password is wrong. On second thought, the backend should probably return 401 as that's a more accurate description of the issue. Fixed.